test/scenarios/el94-base@el94-src@upgrade-ok.sh

```diff
@@ -13,6 +13,5 @@ scenario_remove_vms() {
 
 scenario_run_tests() {
     run_tests host1 \
-        --variable "TARGET_REF:rhel-9.2-microshift-source" \
```

Comment:
Need to update this one to `rhel-9.4-microshift-source`?

Reply:
i've restored the truncated argument and corrected the version to 9.4